src/main/java/com/labsynch/labseer/chemclasses/rdkit/ChemStructureServiceRDKitImpl.java

```diff
@@ -466,56 +399,76 @@ public boolean deleteStructure(StructureType structureType, int cdId) {
 		return true;
 	}
 
+	private String getServiceFormat(String format) {
+		String serviceFormat = null;
+		if (format == null || format.equalsIgnoreCase("mol")) {
```

Comment:
This is a little funny. Are we doing any trimming of `$$$$`? MOL and SDF are very similar but not the same format

Reply:
Look like this function toFormat is barely used outside of implementation specific usage. There is a molConvert route which is likely specific to marvin js which uses this but thats not really applicable here.  I should refactor this to remove it completely from RDkit implementation but I will just do this as a TODO item. (noted).